VSCode/armsnippets.json

```diff
@@ -806,12 +1013,65 @@
     ],
     "description": "DNS Zone"
 },
+"Function": {
+    "prefix": "arm-function",
+    "body": [
+      " {",
+      "      \"apiVersion\": \"2015-08-01\",",
+      "      \"type\": \"Microsoft.Web/sites\",",
+      "      \"name\": \"${1:functionName}\",",
+      "      \"location\": \"[resourceGroup().location]\",",
+      "      \"kind\": \"functionapp\",",
+      "      \"dependsOn\": [",
+      "        \"[resourceId('Microsoft.Web/serverfarms',${2:serverFarmName})]\",",
+      "        \"[resourceId('Microsoft.Storage/storageAccounts', ${3:storageAccountName})]\"",
+      "      ],",
+      "      \"properties\": {",
+      "        \"serverFarmId\": \"[resourceId('Microsoft.Web/serverfarms', ${2:serverFarmName})]\",",
+      "        \"siteConfig\": {",
+      "          \"appSettings\": [",
+      "            {",
+      "              \"name\": \"AzureWebJobsDashboard\",",
+      "              \"value\": \"[concat('DefaultEndpointsProtocol=https;AccountName=', ${3:storageAccountName}, ';AccountKey=', listKeys(${4:storageAccountID},'2015-05-01-preview').key1)]\"",
+      "            },",
+      "            {",
+      "              \"name\": \"AzureWebJobsStorage\",",
+      "              \"value\": \"[concat('DefaultEndpointsProtocol=https;AccountName=', ${3:storageAccountName}, ';AccountKey=', listKeys(${4:storageAccountID},'2015-05-01-preview').key1)]\"",
+      "            },",
+      "            {",
+      "              \"name\": \"WEBSITE_CONTENTAZUREFILECONNECTIONSTRING\",",
+      "              \"value\": \"[concat('DefaultEndpointsProtocol=https;AccountName=', ${3:storageAccountName}, ';AccountKey=', listKeys(${4:storageAccountID},'2015-05-01-preview').key1)]\"",
+      "            },",
+      "            {",
+      "              \"name\": \"WEBSITE_CONTENTSHARE\",",
+      "              \"value\": \"[toLower(${1:functionName})]\"",
+      "            },",
+      "            {",
+      "              \"name\": \"FUNCTIONS_EXTENSION_VERSION\",",
+      "              \"value\": \"~2\"",
+      "            },",
+      "            {",
+      "              \"name\": \"APPINSIGHTS_INSTRUMENTATIONKEY\",",
+      "              \"value\": \"[reference(resourceId('microsoft.insights/components/', variables('applicationInsightsName')), '2015-05-01').InstrumentationKey]\"",
+      "            },",
+      "            {",
+      "              \"name\": \"FUNCTIONS_WORKER_RUNTIME\",",
+      "              \"value\": \"${5|dotnet,node,java|}\"",
+      "            }",
+      "          ]",
+      "        }",
+      "      }",
+      "    }",
+      ""
+    ],
+    "description": "Azure Functions (v2)"
+  },
 "KeyVault": {
     "prefix": "arm-keyvault",
     "body": [
         "{",
         "    \"type\": \"Microsoft.KeyVault/vaults\",",
-        "    \"apiVersion\": \"2016-10-01\",",
+        "    \"apiVersion\": \"2018-02-14\",",
         "    \"name\": \"${1:keyVault1}\",",
```

Comment:
New apiVersion doesn't require any changes to the template?

Reply:
Yes, there are new values, have added these now.